web/tests/lynxkite.ts

```diff
@@ -975,3 +981,10 @@ expect.extend({
     return { pass: true };
   }
 });
+declare global {
+  namespace PlaywrightTest {
+    interface Matchers<R> {
+      toConcur(expected: any): R;
+    }
+  }
+}
```

Comment:
Wow. Didn't know we could do that.

Reply:
> Wow. Didn't know we could do that.

Well `@typescript-eslint/no-namespace` doesn't think we can do it either. I think this sort of type declaration is supposed to be put in a separate file which is then referenced in `tsconfig.json`. But I like putting this next to the definition of `toConcur`. Plus VS Code already can't find our `tsconfig.json`. So I just disabled that rule. 🤷